iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -562,3 +571,133 @@ TEST_F(PortManager_test, PortDestroy)
         }
     }
 }
+
+TEST_F(PortManager_test, CheckNodeOverflowErrorReturnsnullptrForAcquireNodeData)
```

Comment:
@elBoberido   all tests related to node added by me would be removed since i found this related PR update the tests related to node :
https://github.com/eclipse-iceoryx/iceoryx/issues/349